NuGet.Config

```diff
@@ -16,6 +16,6 @@
     <add key="vctools" value="https://vcppdogfooding.azurewebsites.net/nuget/" />
     <add key="myget.org roslyn tools" value="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
     <add key="myget.org roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
-    <add key="artifacts" value="../artifacts" />
+    <add key="artifacts" value="artifacts" />
```

Comment:
artifacts directory is in the same spot. I moved the NuGet.Config out of `.nuget` directory and got rid of `.nuget`. It is normal now to have the NuGet.Config along side the sln files. The `dotnet restore` tasks will automatically find it in the new location.